frontend/src/features/mapSideBar/tabs/ltsa/LtsaLandSubForm.tsx

```diff
@@ -0,0 +1,53 @@
+import { Input } from 'components/common/form';
+import { FieldArray, getIn, useFormikContext } from 'formik';
+import { DescriptionOfLand, LtsaOrders } from 'interfaces/ltsaModels';
+import * as React from 'react';
+import { withNameSpace } from 'utils/formUtils';
+
+import { SectionFieldWrapper } from '../SectionFieldWrapper';
+import { StyledSectionHeader } from '../SectionStyles';
+import LtsaLegalNotationsSubForm from './LtsaLegalNotationsSubForm';
+
+export interface ILtsaLandSubFormProps {
+  nameSpace?: string;
+}
+
+export const LtsaLandSubForm: React.FunctionComponent<ILtsaLandSubFormProps> = ({ nameSpace }) => {
+  const { values } = useFormikContext<LtsaOrders>();
+  const lands = getIn(values, withNameSpace(nameSpace, 'descriptionsOfLand')) ?? [];
+  return (
+    <>
+      <StyledSectionHeader>Land</StyledSectionHeader>
+      {lands.length === 0 ? (
+        'this title has no land'
+      ) : (
+        <>
+          <FieldArray
+            name={withNameSpace(nameSpace, 'descriptionsOfLand')}
+            render={({ name }) => (
+              <React.Fragment key={`land-row-${name}`}>
+                {lands.map((land: DescriptionOfLand, index: number) => {
+                  const innerNameSpace = withNameSpace(nameSpace, `descriptionsOfLand.${index}`);
```

Comment:
would this cause React to complain about the key not being present?

Reply:
Ill double check